streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -235,11 +235,7 @@ public final Sensor threadLevelSensor(final String threadId,
         final String key = threadSensorPrefix(threadId);
         synchronized (threadLevelSensors) {
             final String fullSensorName = key + SENSOR_NAME_DELIMITER + sensorName;
```

Comment:
I think you can also drag this line into `getSensor()`. Sorry for not noticing before.